components/director/internal/domain/application/service.go

```diff
@@ -1057,3 +1142,12 @@ func (s *service) genericUpsert(ctx context.Context, appTenant string, in model.
 
 	return nil
 }
+
+func containsStr(s []string, str string) bool {
```

Comment:
add this function to director/pkg/str package or check if such doesn't already exist there and reuse it